src/app/api/gql/fragments/fragment-dataset-search-overview.graphql

```diff
@@ -1,5 +1,21 @@
 fragment DatasetSearchOverview on Dataset {
-  ...DatasetBasics
+  id
+  kind
+  name
+  owner {
+    ...AccountExtended
```

Comment:
I was surprised to see that `DatasetBasics` fragment was not enough to display the datasets in search results. I assume this is only because `AccountBasics` (which is part of `DatasetBasics`) does not contain `avatarUrl`.

The separation between `AccountBasics` and `AccountExtended` would make sense if the first one was saving some extra database queries on the server side, but `AccountBasics::accountProvider` already makes the server load the full profile information of the account, so requesting `avatarUrl` adds almost zero work.

I would suggest extending `AccountBasics` with `avatarUrl` instead and changing this fragment back to `DatasetBasics`. We can reserve `AccountExtended` for more detailed stuff in the future.

Reply:
I remember that Sergey said that it is better not to just take the `avatarUrl ` if you do not use it (he was talking about productivity). The fragment `DatasetBasics ` is used in many places where the `avatarUrl` is not needed. In my opinion, it is not a very good idea to expand the fragment `DatasetBasics ` with the avatarUrl field.

Reply:
`accountProvider` was added recently. If it is called, there is no extra cost for `avatarUrl`.